examples/pytorch/grand/main.py

```diff
@@ -0,0 +1,198 @@
+import dgl.function as fn
+def GRANDConv(graph, feats, order):
+    
+    with graph.local_scope():
+        
+        degs = graph.in_degrees().float().clamp(min=1)
+        norm = th.pow(degs, -0.5).to(feats.device).unsqueeze(1)
+
+        graph.ndata['norm'] = norm
+        graph.apply_edges(fn.u_mul_v('norm', 'norm', 'weight'))
+
+        x = feats
+        y = 0+feats
+
+        for i in range(order):
+            graph.ndata['h'] = x
+            graph.update_all(fn.u_mul_e('h', 'weight', 'm'), fn.sum('m', 'h'))
+            x = graph.ndata.pop('h')
+            y.add_(x)
+
+    return x
+
+
+import argparse
```

Comment:
In Python coding style, we normally put all import clauses at the very beginning of a script.